### ISSUE TITLE
支持ARM的吗？

### ISSUE THREAD

**Issue by skyformat99** [hamler-lang/hamler#171]
目前支持ARM吗？32  64bit
请教

**Comment by emqplus** [comment]
Yes. The hamler source code will be compiled to Beam bytecode, and running on Erlang VM.

**Comment by Clotho-FP** [comment]
Arm64 binary would be cool so we wouldn't have to install stack
